src/modules/getBullsAndCows.js

```diff
@@ -13,7 +13,22 @@
  * Example: { bulls: 1, cows: 2 }
  */
 function getBullsAndCows(userInput, numberToGuess) {
-  /* Write your code here */
+  const input = userInput.toString();
+  const number = numberToGuess.toString();
+  const result = {
+    bulls: 0,
+    cows: 0,
+  };
+
+  for (let i = 0; i < input.length; i++) {
+    if (input[i] === number[i]) {
```

Comment:
The for loop is used here, but it's better to use Array.prototype.forEach for such operations.